services/galley/src/Galley/Cassandra/Queries.hs

```diff
@@ -360,6 +360,11 @@ rmMemberClient c =
   let t = LT.fromStrict (client c)
    in QueryString $ "update clients set clients = clients - {'" <> t <> "'} where user = ?"
 
+-- MLS Clients --------------------------------------------------------------
+
+addMLSClients :: PrepQuery W (ConvId, UserId, C.Set ClientId) ()
+addMLSClients = "update member set mls_clients = mls_clients + ? where conv = ? and user = ?"
```

Comment:
Doesn't the order of types in the tuple in the signature matter, i.e., shouldn't you use corresponding values in the same order? Here it looks like you're doing a + operation (which I guess is set union in Cassandra) between a Set of client IDs and a conversation ID.

Reply:
Also fixed later.